.github/workflows/check-patch.yml

```diff
@@ -0,0 +1,105 @@
+name: Check patch
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-el8:
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream8
+
+    steps:
+    - name: prepare env
+      run: |
+           mkdir -p ${PWD}/tmp.repos/BUILD
+           yum install -y --setopt=tsflags=nodocs autoconf automake gettext-devel git systemd make git rpm-build
+
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: autoreconf
+      run: autoreconf -ivf
+
+    - name: configure
+      run: ./configure
+
+    - name: run distcheck
+      run: make -j distcheck
+
+    - name: Build RPM
+      run: rpmbuild -D "_topdir ${PWD}/tmp.repos" -D "release_suffix .$(date -u +%Y%m%d%H%M%S).git$(git rev-parse --short HEAD)" -ta ovirt-release*.tar.gz
+
+    - name: Collect artifacts
+      run: |
+          mkdir -p exported-artifacts
+          find tmp.repos -iname \*rpm -exec mv "{}" exported-artifacts/ \;
+          mv ./*tar.gz exported-artifacts/
+
+    - name: test install
+      run: |
+          yum install -y exported-artifacts/ovirt-release-master-4*noarch.rpm
+          yum module enable -y javapackages-tools:201801
+          yum module enable -y maven:3.5
+          yum module enable -y pki-deps:10.6
+          yum module enable -y postgresql:12
+          yum --downloadonly install -y exported-artifacts/*noarch.rpm
```

Comment:
Why downloadonly? Perhaps comment. What is it good for?

Reply:
no real need to test if the whole project is working here, just checking that the provided repos are enough for installing the project.